README.md

```diff
@@ -1,4 +1,5 @@
 # Helix
+> This is for ***users*** only. If you are interested in contributing, have a look at [CONTRIBUTING.md](./CONTRIBUTING.md). If you have a doubt, consider asking it in [discussions](https://github.com/helix-editor/helix/discussions) or [Matrix Space](https://matrix.to/#/#helix-community:matrix.org)(Matrix Space is more active).
```

Comment:
There's a contributing section towards the end of the file, I think it's a better fit there:

https://github.com/helix-editor/helix/blob/eba0bbda2ee10e9ba53f01f1d324d2c4c82229d2/README.md#L71-L75

Reply:
I just put it so that if anyone has a doubt or a question, they need not go till the end of the README. Its mentioned what and where at the top with links, so that they can just immediately go, rather than searching about that, and create an issue just for a question...

Reply:
I agree with @sudormrfbin, drop this entire section. I think it's common to put it at the end -- if you're new to the project you'd like to see an explanation on what this is first, rather than where the contribution guide is.